src/backend/gl/src/queue.rs

```diff
@@ -612,7 +612,7 @@ impl CommandQueue {
                     r.image_extent.height as _,
                     glow::RGBA,
                     glow::UNSIGNED_BYTE,
-                    0,
+                    r.buffer_offset as i32,
```

Comment:
we should use the offset in `CopyTextureToBuffer` as well

Reply:
Whoops!  I did in fact forget about `CopyTextureToBuffer`, and it's now waiting on https://github.com/grovesNL/glow/pull/13

Reply:
Done, but it requires depending on a git rev of glow again.